stdlib/streams/pom.xml

```diff
@@ -70,6 +70,26 @@
             <groupId>org.ballerinalang</groupId>
             <artifactId>ballerina-crypto</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Add these dependencies to .gradle file as well.

Reply:
Added in d666ea1